src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1521,7 +1521,10 @@ static void FillScanInfo(Eventinfo *lf,cJSON *scan_id,cJSON *name,cJSON *descrip
             sprintf(value, "%d", scan_id->valueint);
         } else if (scan_id->valuedouble) {
             sprintf(value, "%lf", scan_id->valuedouble);
-        } 
+        } else {
+            return;
```

Comment:
This is an unexpected condition: invalid value. Please print a debugging log here.